inc/server/class-form-server.php

```diff
@@ -7,6 +7,8 @@
 
 namespace ThemeIsle\GutenbergBlocks\Server;
 
+use PHP_CodeSniffer\Reports\Json;
```

Comment:
Why exactly? PHP code sniffer is a dev-dependency so it won't be present in production. Have you tested to make sure no error is thrown there?